flow/project.py

```diff
@@ -1041,13 +977,12 @@ def _print_progress(x):
     ])
     "Pretty (unicode) symbols denoting the execution status of operations."
 
-    @_support_legacy_api
     def print_status(self, jobs=None, overview=True, overview_max_lines=None,
                      detailed=False, parameters=None, skip_active=False, param_max_width=None,
                      expand=False, all_ops=False, only_incomplete=False, dump_json=False,
                      unroll=True, compact=False, pretty=False,
                      file=None, err=None, ignore_errors=False,
-                     no_parallelize=False):
+                     no_parallelize=False, full=False):
```

Comment:
Instead of introducing another (redundant) argument to the Python API, we should instead resolve the ``--full`` option as part of the argument parser handling. See [line 2622](https://github.com/glotzerlab/signac-flow/blob/98d64c83263864011b0939d1b1fc2ab6c9077739/flow/project.py#L2622) for an example.